scripts/generate-component-config/write-component-config.js

```diff
@@ -44,7 +44,7 @@ const writeComponentConfig = (packageConfigs, componentImports, outputPath, addP
     generatedConfig = generatedConfig.replace('<test_imports>', imports.tests);
     generatedConfig = generatedConfig.replace('<config>', `const componentConfig = ${configString};`);
 
-    fs.writeFileSync(path.join(path.resolve(process.cwd(), outputPath), 'generatedComponentConfig.js'), generatedConfig);
+    fs.writeFileSync(path.join(path.resolve(process.cwd(), outputPath), 'generatedComponentConfig.jsx'), generatedConfig);
```

Comment:
Does the generated component config actually have jsx in it?

Reply:
The generateComponentConfig output: https://github.com/cerner/terra-framework/blob/b0ad88a3f3d01025f2582c8dca748485d6db2526/generatedComponentConfig.jsx

Reply:
Looks like it does not.

Reply:
Lets leave any file without jsx syntax in it marked as .js